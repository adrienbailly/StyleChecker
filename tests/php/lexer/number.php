7
1234567
1.23
0.123
.123
1.2e3
7E-10
