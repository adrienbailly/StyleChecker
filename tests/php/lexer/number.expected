(ICONST,'7',1,0)
(NEWLINE,'\n',1,1)
(ICONST,'1234567',2,2)
(NEWLINE,'\n',2,9)
(FCONST,'1.23',3,10)
(NEWLINE,'\n',3,14)
(FCONST,'0.123',4,15)
(NEWLINE,'\n',4,20)
(PERIOD,'.',5,21)
(ICONST,'123',5,22)
(NEWLINE,'\n',5,25)
(FCONST,'1.2e3',6,26)
(NEWLINE,'\n',6,31)
(ICONST,'7',7,32)
(ID,'E',7,33)
(MINUS,'-',7,34)
(ICONST,'10',7,35)
(NEWLINE,'\n',7,37)
